### PR TITLE
OS-250 Removing JSONArray dependency in ArrayHelper.java and Fixing S…

### PR DESCRIPTION
**Pull request by reshmi-nair**
…ignature issue with viewTemplate

### CODE REVIEW

File: java/view-templates/src/main/java/io/opensaber/views/ViewTransformer.java [hunk 1]
```diff
@@ -34,15 +35,18 @@ public JsonNode transform(ViewTemplate viewTemplate, JsonNode node) throws Excep
 				ArrayNode resultArray = JsonNodeFactory.instance.arrayNode();
 
 				for (int i = 0; i < nodeAttrs.size(); i++) {
+					
 					JsonNode tNode = tranformNode(viewTemplate, nodeAttrs.get(i));
+					appendSignatures(viewTemplate, nodeAttrs.get(i),tNode);
```
Comment (indrajra) [inline java/view-templates/src/main/java/io/opensaber/views/ViewTransformer.java]: Move appendSignatures inside transformNode() .

File: java/view-templates/src/main/java/io/opensaber/views/ViewTransformer.java [hunk 2]
```diff
@@ -97,4 +101,32 @@ private JsonNode tranformNode(ViewTemplate viewTemplate, JsonNode nodeAttrs) thr
         }
         return result;
     }
-}
+    
+    private JsonNode appendSignatures(ViewTemplate viewTemplate, JsonNode nodeAttrs, JsonNode resultNode) throws Exception {
+    	if(nodeAttrs.get("signatures")!=null) {
+			
+			ArrayNode sigArray = new ObjectMapper().createArrayNode();
```
Comment (indrajra) [inline java/view-templates/src/main/java/io/opensaber/views/ViewTransformer.java]: should we iterate or just would be able to say like resultNode['signatures'] = nodeAttrs.get['signatures'] ?

Comment (reshmi-nair) [reply java/view-templates/src/main/java/io/opensaber/views/ViewTransformer.java]: iteration is to specifically find out the signatures of fields specified in viewtemplate.

File: java/view-templates/src/main/java/io/opensaber/views/ViewTransformer.java (hunk 1 above)
Comment (reshmi-nair) [reply java/view-templates/src/main/java/io/opensaber/views/ViewTransformer.java]: this is done.

**Pull request merged**